files/en-us/web/accessibility/understanding_colors_and_luminance/index.html

```diff
@@ -2,289 +2,502 @@
 
-<p>Luminance and lightness are not the same.</p>
+<img src="colorcomb03.png" alt="the previous examples, but adding green to the brightest color to improve contrast.">
 
-<p>Luminance and reflected light are not the same.</p>
+<p>Thus it is always important to ensure ample luminance contrast. This is most easily regulated by adding green to the brightest of the two colors.</p>
 
-<h2 id="See_also">See also</h2>
+<h3 id="Working_with_Red">Working with Red</h3>
 
-<h4 id="MDN"><strong>MDN</strong></h4>
+<p>Pure red (<code>#F00</code>) text on a pure black (<code>#000</code>) background is a bare minimum contrast for a 24px 700 weight bold font for normal vision. Someone with protanopia would have a difficult time reading this on an sRGB monitor, and it would be literally invisible on a next-generation Rec2020 display.</p>
+
+<img src="colorcomb04.png" alt="pure red on black, and how someone with protanopia sees it, which is reduced to being illegible.">
+
+<p>Here again, adding a small amount of green (#77) makes an noticeable difference for protanopia, and in fact improves readability for all users by improving the luminance contrast.</p>
+
+<img src="colorcomb05.png" alt="the previous example, but adding a small amount of green to increase the luminance contrast of the text against black.">
+
+<p>There are <a href="https://www.myndex.com/CVD/">color vision simulators</a>, that allow you to process a screenshot of your design to demonstrate how your design might look to someone with a color vision deficiency.</p>
 
-<ul>
- <li><a href="/en-US/docs/Web/Accessibility">Accessibility</a></li>
- <li><a href="/en-US/docs/Learn/Accessibility">Accessibility learning path</a></li>
- <li><a href="/en-US/docs/Web/CSS/color">color</a></li>
- <li><a href="/en-US/docs/Web/CSS/color_value">&lt;color&gt;</a></li>
- <li><a href="/en-US/docs/Web/Accessibility/Seizure_disorders">Web accessibility for seizures and physical reactions</a></li>
- <li>Color blindness</li>
-</ul>
 
-<h4 id="Discussions_General"> Discussions (General)</h4>
+<p>An excellent guide regarding designing with color for accessibility is Maureen Stone's <a href="https://www.nist.gov/system/files/documents/itl/vote/NISTIR-7537.pdf">Guidelines for Using Color</a> a free download published by NIST.</p>
+
+
+<h2 id="Psychophysical_aspects_of_color">Psychophysical aspects of color</h2>
+
+<p>Color as in hues and saturation can affect our mood, and enhance — or de-enhance — our interactive experiences.</p>
+
+
+<h3 id="Examples_of_colors_effect_beyond_vision">Examples of color's effect beyond vision</h3>
 
 <ul>
- <li><a href="https://ask.metafilter.com/312049/What-is-the-grayscale-setting-for-in-accessibility-options">What is the “grayscale” setting for in accessibility options?</a></li>
+  <li>Color can be culturally dependent</li>
+  <ul>
+    <li><a href="https://journals.sagepub.com/doi/10.1177/002202217300400201">A Cross-Cultural Study of the Affective Meanings of Color</a></li>
+  </ul>
```

Comment:
```suggestion
  <li><strong>Color can be culturally dependent:</strong> <a href="https://journals.sagepub.com/doi/10.1177/002202217300400201">A Cross-Cultural Study of the Affective Meanings of Color</a></li>
```

I think there might have been a miscommunication here, and that's on me. I'm going to make suggestions for each bullet point to not have a nested list consisting of a single item, each following the "<strong>Concept:</strong> source" pattern.